dailydotdev/docs
# Fix: old docs reputation route

Hello everyone! 
I hope this pr can be useful, forgive me If I made any mistakes.

_The problem_:
There is this url (https://docs.daily.dev/docs/how-does-daily-dev-work/reputation) that is still being indexed from Google, so we need some sort of redirect to avoid a 404 page.
I've found that the `plugin-client-redirects` is useful for this case, as it can correctly redirect the page.
**Please note** that the plugin only works under production environment.

_How to test it:_
Run `npm run build` && `npm run serve`
Go on `/docs/how-does-daily-dev-work/reputation`
You should be able to see the redirect.

In order to use the plugin, I've updated Docusaurus to its latest version.
